Dockerfile.heroku-24

```diff
@@ -0,0 +1,50 @@
+FROM gliderlabs/herokuish:v0.10.3-24
+
+# Add perl buildpack for morph
+RUN /bin/herokuish buildpack install https://github.com/miyagawa/heroku-buildpack-perl.git 1f7fafa95a00ee39df9d5a035c6a253d1d79fe56
+
+# Add certificate authority used by mitmproxy
+# Also needs to be identical to the cert at mitmproxy/mitmproxy-ca-cert.pem in
+# https://github.com/openaustralia/morph
+ADD mitmproxy-ca-cert.pem /usr/local/share/ca-certificates/mitmproxy-ca-cert.crt
+RUN update-ca-certificates
+
+# Add prerun script which will disable output buffering for ruby
+ADD prerun.rb /usr/local/lib/prerun.rb
+
+# poppler-utils has a more recent pdftohtml than the pdftohtml package
+# pdftohtml is needed by the python scraperwiki library
+# libffi-dev needed by python cffi
+# time is needed directly by morph.io for scraper run measurements
+RUN apt-get update && apt-get install -y time libblas-dev liblapack-dev gfortran swig protobuf-compiler libprotobuf-dev libsqlite3-dev poppler-utils libffi-dev
+
+# PhantomJS has been deprecated
+# RUN apt-get install -y phantomjs
```

Comment:
I *think* that the use of PhantomJS is probably obsoleted by the use of chromedriver. I think this would be an issue for backwards compatibility on an existing platform; but since heroku-24 is a new platform, I'm guessing it's probably okay to just say that using heroku-24 means that phantomjs is no longer available?